app/assets/stylesheets/shared/form.scss

```diff
@@ -1,5 +1,21 @@
 // material style floating form labels
 .form-floating {
+  display: flex;
+  align-items: center;
+  border: 1px solid $gray-200;
+  border-radius: 0.1875rem;
```

Comment:
I had to move form styling to the wrapper so the helptext could slide right in. It won't affect fields without help text! Flexbox FTW.